lib/proxi/local/sdrf_handler.py

```
@@ -0,0 +1,729 @@
+            icolumn += 1
+
+        for row in sdrf_data['rows']:
+            for title in columns_of_interest:
+                if column_indexes[title] is not None:
+                    icolumn = column_indexes[title]
+                    column_unique_values[title][row[icolumn]] = True
+
+        #### Record which columns are completely empty (i.e. no values in the column)
+        for row in sdrf_data['rows']:
+            icolumn = 0
+            for title in titles:
```

Ruff (0.15.12)

[warning] Loop control variable `title` not used within loop body

(B007)